recipes/pipestat/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set name = "pipestat" %}
+{% set version = "0.5.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pipestat-{{ version }}.tar.gz
+  sha256: 986c6edd9097b22fec445c0dcb5283a4d992b9942e6df49857bf57beef5c9792
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  entry_points:
+    - pipestat = pipestat.__main__:main
+
+requirements:
+  host:
+    - python >=3.8
+    - pip
+  run:
+    - python >=3.8
+    - setuptools
+    - jsonschema
+    - logmuse >=0.2.5
+    - oyaml
+    - psycopg2-binary
```

Comment:
`psycopg2-binary` is a convenient wheel when using pip. With conda, I'd prefer to specify `psycopg2` directly.

Will work with `psycopg2-binary` as well. It's just an alternative name (will install `psycopg2`).

Reply:
Thank you for catching this, I have corrected psycopg2-binary to psycopg2 in my next commit.